examples/pylint/C0301_line_too_long.py

```diff
@@ -0,0 +1 @@
+TEMP = "I am in the file C0301_line_too_long.py. I raise the error for a very long line."
```

Comment:
I agree with @nigef. Also don't use the term "raise" for making pylint emit errors, since this is different from the program raising an exception.